packages/modules/desktop-viewer-react/package.json

```diff
@@ -85,7 +85,7 @@
     "typescript": "^3.7.2"
   },
   "dependencies": {
-    "@itwin/viewer-react": "1.0.1"
+    "@itwin/viewer-react": "workspace:*"
```

Comment:
is this right? We are publishing this package and not just using this in the monorepo